src/main/java/spoon/compiler/Environment.java

```diff
@@ -210,16 +210,6 @@ void report(Processor<?> processor, Level level,
 	int getWarningCount();
 
 	/**
-	 * Gets the class loader used to compile/process the input source code.
-	 */
-	ClassLoader getInputClassLoader();
-
-	/**
-	 * Sets the class loader used to compile/process the input source code.
-	 */
-	void setInputClassLoader(ClassLoader classLoader);
```

Comment:
I'm a bit concerned about removing this method that I'm using. The idea is that when you embedded this Spoon as a third-party framework in another framework, you need a simple and direct way to instruct Spoon the class loader that you want to use and not bother about some complicated stuff with class paths and thread context class loader. So I would vote against removing this method.